lib/brains/views/method.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.errors import BraintreeResultError
+from lib.brains.forms import PaymentMethodForm
+from lib.brains.serializers import (
+    PaymentMethodSerializer as BraintreeSerializer)
+from lib.buyers.models import PaymentMethod
+from lib.buyers.serializers import PaymentMethodSerializer
+from lib.transactions.constants import PROVIDER_BRAINTREE
+from solitude.constants import PAYMENT_METHOD_CARD
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains')
+
```

Comment:
Why can't you just define a GET endpoint here that queries ``PaymentMethod``? I think it would avoid confusion since it's weird that you can't post to the generic PaymentMethod endpoint. I also think it's perhaps overly optimistic that we will use the concept of payment methods for other providers.